# DepMap Public 23Q2

##############
## Overview ##
##############

This DepMap release contains data from CRISPR knockout screens from project Achilles, as well as genomic characterization data from the CCLE project.

###############
## Pipelines ##
###############

### Achilles

This Achilles dataset contains the results of genome-scale CRISPR knockout screens for Achilles (Avana Cas9 library) and Achilles combined with Sanger's Project SCORE (KY Cas9 library) screens. The dataset was processed using the following steps:
- Sum raw readcounts by SequenceID and collapse pDNAs by median (removing sequences with fingerprinting issues, total reads below 500000 or less then 25 mean reads per guide)
- Normalize readcounts such that pDNA sequences have the same mode, and all other sequences have median of nonessentials that matches the median of nonessentials in the corresponding pDNA batch
- Remove intergenic/non-targeting controls, Cas9 guides with multiple alignments to a gene, guides with >5 genomic alignments, and sgRNAs with inconsistent signal (see DropReason of guide maps).
- NA sgRNAs with pDNA counts less than one millionth of the pDNA pool
- Calculate the mean reads per guide for each sequence. Sequences with more then 185 mean reads per guide are considered passing.
- Compute the log2-fold-change (LFC) from pDNA counts for each sequence. Collapse to a gene-level by taking the median of guides.
- Calculate the Null-Normalized Median Difference (NNMD) from the LFC using the following equation: (median(essentials) - medain(nonessentials)) / MAD(nonessentials).  Sequences must be below a threshold of -1.25 to be considered passing.
- Remove sequences that do not have a Pearson coefficient > .41 with at least one other sequence for the screen when looking at genes with the highest variance in gene effect across screens and libraries.
- Calculate the NNMD for each screen after averaging passing sequences. Screen must be below a threshold of -1.25 to be considered passing.
- NA out readcounts for pDNAs which correspond with a SNP in any guide in a vector for a given cell line (to correct ancestry bias), then recompute LFC.
- Compute the naive gene score by collapsing the passing LFC data to a screen x gene matrix.
- Prior to running Chronos, NA apparent outgrowths in readcounts.
- Run Chronos per library-screen type to generate screen-level gene effect scores, apply copy number correction, then scale such that median of common essentials is at -1.0 and median of nonessentials is at 0.  Concatenate gene effects from all libraries into a single ScreenGeneEffect matrix.
- Map all of the screen-level gene effects to their corresponding models, then run Harmonia on the Avana and KY to correct the batch effect in Cas9 data. Produces an integrated model-level CRISPRGeneEffect matrix.
- Using the CRISPRGeneEffect, identify pan-dependent genes as those for whom 90% of cell lines rank the gene above a given dependency cutoff. The cutoff is determined from the central minimum in a histogram of gene ranks in their 90th percentile least dependent line.
- For each Chronos gene effect score from both the ScreenGeneEffect and CRISPRGeneEffect, infer the probability that the score represents a true dependency. This is done using an EM step until convergence independently in each screen/model. The dependent distribution is given by the list of essential genes. The null distribution is determined from unexpressed gene scores in those cell lines that have expression data available, and from the nonessential gene list in the remainder.
The essential  and nonessential controls used throughout the analysis are the Hart reference nonessentials and the intersection of the Hart and Blomen essentials. See Hart et al., Mol. Syst. Biol, 2014 and Blomen et al., Science, 2015. Lists of these genes are provided as AchillesCommonEssentialControls.csv and AchillesNonessentialControls.csv.
Note that the source for copy number data varies by cell line. Copy number data indicated as 'Sanger WES' are based on the Sanger Institute whole exome sequencing data (COSMIC: http://cancer.sanger.ac.uk/cell_lines, EGA accession number: EGAD00001001039) reprocessed using CCLE pipelines. Copy number source was chosen according to the following logic:
- Broad WES for lines where available
- Broad SNP when Broad WES is not available and Sanger WES not available, or Sanger WES copy number has less correlation with logfold change than Broad SNP
- Sanger WES in all other cases

### Expression

DepMap expression data is quantified from RNAseq files using the GTEx pipelines. A detailed description of the pipelines and tool versions can be found here: https://github.com/broadinstitute/ccle_processing#rnaseq. We provide a subset of the data files outputted from this pipeline available on FireCloud. These are aligned to hg38.

### Copy number

DepMap WES copy number data is generated by running the GATK copy number pipeline aligned to hg38. Tutorials and descriptions of this method can be found here https://software.broadinstitute.org/gatk/documentation/article?id=11682, https://software.broadinstitute.org/gatk/documentation/article?id=11683. WES samples have been realigned to hg38 and run through this pipeline.

### Mutations

DepMap mutation calls are generated using Mutect2 and annotated and filtered downstream. Variants are aligned to hg38. Detailed documentation can be found here https://storage.googleapis.com/shared-portal-files/Tools/%5BDMC%20Communication%5D%2022Q4%20Mutation%20Pipeline%20Update.pdf.

### Fusions

DepMap generates RNAseq based fusion calls using the STAR-Fusion pipeline. A comprehensive overview of how the STAR-Fusion pipeline works can be found here: https://github.com/STAR-Fusion/STAR-Fusion/wiki. We run STAR-Fusion version 1.6.0 using the plug-n-play resources available in the STAR-Fusion docs for gencode v29. We run the fusion calling with default parameters except we add the --no_annotation_filter and --min_FFPM 0 arguments to prevent filtering.

###########
## Files ##
###########

### README.txt

Description of all files contained in this release

### CRISPRGeneEffect.csv

Pipeline: Achilles

_Post-Chronos_
Gene effect estimates for all models, integrated using Harmonia.
- Columns: Gene - Rows: ModelID

### CRISPRGeneDependency.csv

Pipeline: Achilles

_Post-Chronos_
Gene dependency probability estimates for all models in the integrated gene effect.
- Columns: Gene - Rows: ModelID

### OmicsExpressionProteinCodingGenesTPMLogp1.csv

Pipeline: Expression

Gene expression TPM values of the protein coding genes for DepMap cell lines. Values are inferred from RNA-seq data using the RSEM tool and are reported after log2 transformation, using a pseudo-count of 1; log2(TPM+1).
Additional RNA-seq-based expression measurements are available for download as part of the full DepMap Data Release
More information on the DepMap Omics processing pipeline is available at <https://github.com/broadinstitute/depmap_omics>.

### OmicsCNGene.csv

Pipeline: Copy number

Gene-level copy number data that is log2 transformed with a pseudo-count of 1; log2(CN ratio + 1) . Inferred from WGS, WES or SNP array depending on the availability of the data type. Values are calculated by mapping genes onto the segment level calls and computing a weighted average along the genomic coordinate.
Additional copy number datasets are available for download as part of the full DepMap Data Release.
More information on the DepMap Omics processing pipeline is available at <https://github.com/broadinstitute/depmap_omics>.

### OmicsSomaticMutations.csv

Pipeline: Mutations

MAF-like file containing information on all the somatic point mutations and indels called in the DepMap cell lines. The calls are generated from Mutect2. A description of the various columns is in the DepMap Release README file.
Additional processed mutation matrices containing genotyped mutation calls are available for download as part of the full DepMap Data Release.
More information on the DepMap Omics processing pipeline is available at <https://github.com/broadinstitute/depmap_omics>.
More information on the mutation pipeline and descriptions of the columns in the MAF can be found here: <https://storage.googleapis.com/shared-portal-files/Tools/%5BDMC%20Communication%5D%2022Q4%20Mutation%20Pipeline%20Update.pdf>

### OmicsProfiles.csv

Profile ID mapping information
Columns: ProfileID, ModelID, ModelConditionID, Datatype, and WESKit
Rows: ProfileID

### OmicsDefaultModelProfiles.csv

indicates which profiles are selected in the model-level datasets
Columns: ModelID, ProfileID, and ProfileType (dna/rna/CN only)

### OmicsDefaultModelConditionProfiles.csv

indicates which profiles are selected for each model condition
Columns: ModelConditionID, ProfileID, and ProfileType (dna/rna/CN only)

### Model.csv

Metadata describing all cancer models/cell lines which are referenced by a dataset contained within the DepMap portal.
Columns:
- ModelID: Static primary key assigned by DepMap to each cell model
- PatientID: Indicates relationships of patients from which a model was derived
- CellLineName: Original cell line name, including punctuation
- StrippedCellLineName: Cell line name with alphanumeric characters only
- Age: Age of tissue donor at time of sample collection, if known
- SourceType: Source of the cell model used by DepMap
- SangerModelID: The corresponding Sanger Model ID for this model
- RRID: Cellosaurus research resource identifier
- DepmapModelType: Internal model type used by DepMap project (Identical to OncotreeCodes but with additional codes added for different types of 'normal' tissue)
- AgeCategory: Classification of age of patient into 'Adult', 'Unknown', 'Pediatric' or 'Fetus'
- GrowthPattern: Typical growth pattern of the cell model when it was onboarded
- LegacyMolecularSubtype: Historical molecular subtype annotations (Annotated prior to adoption of Oncotree)
- PrimaryOrMetastasis: Indicates whether tissue sample is from primary or metastatic site
- SampleCollectionSite: Tissue collection site
- Sex: Sex of tissue donor, if known
- SourceDetail: Details of the source of the cell model used by DepMap
- LegacySubSubtype: Historical sub-subtype annotations (Annotated prior to adoption of Oncotree)
- CatalogNumber: The catalog number for lines obtained from a commercial vendor
- CCLEName: Previous naming system that used the stripped cell line name followed by the lineage; no longer assigned to new cell lines
- COSMICID: Cell line ID used in Cosmic cancer database
- PublicComments: Free text notes about the Model
- WTSIMasterCellID: ID of corresponding record in Sanger Drug dataset
- EngineeredModel: Indicates a genetically modified cell model
- TreatmentStatus: Status of patient treatment at the time the model was derived from tissue. Either 'Unknown', 'Post-treatment' or 'Pre-treatment'
- OnboardedMedia: The media that was initially used to expand cell lines.
- PlateCoating: The type of coating on a plate when a cell model is grown, if any
- OncotreeCode: Oncotree classification of disease (Nomenclature taken from http://oncotree.mskcc.org)
- OncotreeSubtype: Intermediate Oncotree classification (More specific than OncotreeLineage, but often less specific than OncotreeCode)
- OncotreePrimaryDisease: The disease name corresponding to OncotreeCode
- OncotreeLineage: Name of top level tissue from Oncotree nomenclature 

### ModelCondition.csv

The conditions under which the model was assayed.
Columns:
- ModelConditionID: Static primary key assigned by DepMap to each model condition
- Drug: If populated, a compound which the model was treated with in the context of the assay
- CellFormat: Growth pattern of the cell model after it was onboarded
- ModelID: The ID of the associated model/cell line
- Source: Whether the assay was performed at the Broad or Sanger
- GrowthMedia: The media the model was grown in over the course of the assay
- PlateCoating: The type of coating on a plate when a cell model is grown, if any